test/apps/iodemo/io_demo.cc

```diff
@@ -113,7 +113,11 @@ class IoDemoRandom {
     static inline int rand(int min = std::numeric_limits<int>::min(),
                            int max = std::numeric_limits<int>::max()) {
         _seed = (_seed * _A + _C) & _M;
-        return (int)_seed % (max - min + 1) + min;
+        if (max - min == 1) {
+            return _seed & 0x100 ? max : min;
```

Comment:
1. ( ) around _seed & 0x100
2. pls add comment to explain why it's needed: that LCG returns alternating even/odd values

Reply:
@yosefe Done. Thanks!